Riot/Categories/MXSession+Riot.m

```diff
@@ -65,6 +65,7 @@ - (MXHTTPOperation*)vc_canEnableE2EByDefaultInNewRoomWithUsers:(NSArray<NSString
     }
     else
     {
+        MXLogInfo(@"[MXSession] E2EE is disabled by default on this homeserver");
```

Comment:
Can you use MXLogWarning?
It may be useful to log `MXWellknown.JSONDictionnary`.